requirements/dev.txt

```diff
@@ -6,3 +6,4 @@ django-debug-panel==0.8.3
 django-debug-toolbar==1.9.1
 sphinx-autobuild
 django-rest-swagger==2.2.0
+coreapi==2.3.3
```

Comment:
Is this used?

Reply:
Cool! Was just searching the text of the PR and didn't see it mentioned anywhere else.

Btw. thinking of searching... have you done a pure-text search of the repo to check for occurrences of url resolution? I don't have any straight up advice for search strings, but when I have done refactors like this that pertain HTML, JS and Python code, I often just searched for lots of strings with regexxxer (a search tool in the Ubuntu repos)

Reply:
Yeah, I did a combination of pure text searches and regex searches. I am pretty confident that I found everything :)